frameworks/cassandra/universe/config.json

```diff
@@ -108,7 +108,28 @@
               }
             }
           }
-        }
+        },
+        "readiness_check" : {
+          "description" : "Readiness check settings",
```

Comment:
The descriptions should probably contain a warning that messing with these values can lead to spurious failures of the tasks in the framework.